drivers/dai/nxp/sai/sai.h

```diff
@@ -210,6 +210,12 @@ LOG_MODULE_REGISTER(nxp_dai_sai);
 	((dir) == DAI_DIR_RX ? ((UINT_TO_I2S(regmap))->RCSR & (which)) : \
 	 ((UINT_TO_I2S(regmap))->TCSR & (which)))
 
+/* used to clear status flags */
+#define SAI_TX_RX_STATUS_CLEAR(dir, regmap, which)\
+	((dir) == DAI_DIR_RX ?\
+	 SAI_RxClearStatusFlags(UINT_TO_I2S(regmap), which) :\
+	 SAI_TxClearStatusFlags(UINT_TO_I2S(regmap), which))
+
 /* used to retrieve the SYNC direction. Use this macro when you know for sure
```

Comment:
@LaurentiuM1234  change looks good to me. Will approve once this warning is fixed.